nucypher/blockchain/eth/sol/source/contracts/Issuer.sol

```diff
@@ -10,7 +10,8 @@ import "contracts/lib/AdditionalMath.sol";
 
 /**
 * @notice Contract for calculate issued tokens
-**/
+* @dev |v1.1.2|
```

Comment:
What about considering `v.1.1.1` for all contracts?

Reply:
I just tried to remember how many changes we did before, but really it doesn't matter because of testnet redeploy and new versions `v2.1.1`

Reply:
👍